### ISSUE TITLE
Distance from a suburb is useless

### ISSUE THREAD

**Issue by equivalentideas** [JasonThomasData/DocLink#23]
If someone enters "Fairfield, NSW", we don't know there actual location close enough to provide such detailed distance information for the doctor results like "55m", "105m".

Should we just not show it unless we have an actual location point?

![screen shot 2016-05-22 at 12 04 38 pm](https://cloud.githubusercontent.com/assets/1239550/15451722/e02c637c-2015-11e6-87a2-aa258c8cf097.png)

**Comment by JasonThomasData** [comment]
Yeah it's not a necessary feature is it? Is this something you want to remove? I'm happy with that.
Also, if we do go with the HTML5 geolocate option, it's not always going to be super accurate.
We can expect that to be accurate to the nearest suburb, especially in less dense areas.

**Comment by equivalentideas** [comment]
> Yeah it's not a necessary feature is it? Is this something you want to remove?

Yeah I'll remove it for now since we don't have granular location stuff yet to make use of it.
